backend/api/settings.py

```diff
@@ -68,9 +70,11 @@
 
 GRAPHENE = {
     "SCHEMA": "api.graphql_schema.schema",
+    "SCHEMA_OUTPUT": "api.schema.graphql",
```

Comment:
What does this line do / why is it needed for sentry?

Reply:
Need to get more comments in the settings file, I have no clue why I added this.
https://docs.graphene-python.org/projects/django/en/latest/settings/
Does not appear to be relevant, removing.